buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidTestRuleComposer.groovy

```diff
@@ -40,6 +41,8 @@ final class AndroidTestRuleComposer extends AndroidBuckRuleComposer {
             providedDeps.add(RetrolambdaUtil.getRtStubJarRule())
         }
 
+        String manifest = target instanceof AndroidAppTarget ? ":${AndroidBuckRuleComposer.manifest(target)}" : fileRule(target.manifest)
```

Comment:
Why the if/else? Android lib targets can merge manifests too. We should use the merged manifest always

Reply:
I did this for perf savings since the case for using a merged manifest in a lib module test is less convincing. Happy to change for consistency though